libs/domain/picking/src/routes.ts

```diff
@@ -10,6 +13,9 @@ export const defaultPickingRoutes: RouteConfig[] = [
     path: '/picking-list/picking/:id',
     render: ({ id }) =>
       html`<oryx-picking .pickingListId="${id}" mode-light></oryx-picking>`,
+    leave: (): Observable<boolean> => {
+      return resolve(RouterService).shouldGuardRoute();
```

Comment:
Why we delegate this to router service? 
Shouldn't leave callback of specific have all the info it needs to decide, if we should lave (or not) this route?